From 55da812d3a2e76f89f88611381cce8631c0e10bd Mon Sep 17 00:00:00 2001
From: Steve Kondik <steve@cyngn.com>
Date: Fri, 1 May 2015 18:32:14 -0400
Subject: [PATCH 088/173] Camera2: Only autofocus before a snap if we are
 actually in "auto" mode.

Original commit by cyanogen 7daaa72d35c1f8820117768e4b129acd030bcefb

Change-Id: I51853913f6459b599176910f7f486f59972eaf36
---
 src/com/android/camera/FocusOverlayManager.java | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/src/com/android/camera/FocusOverlayManager.java b/src/com/android/camera/FocusOverlayManager.java
index cdf84825f..a03163af5 100644
--- a/src/com/android/camera/FocusOverlayManager.java
+++ b/src/com/android/camera/FocusOverlayManager.java
@@ -626,10 +626,7 @@ public class FocusOverlayManager {
     }
 
     private boolean needAutoFocusCall() {
-        String focusMode = getFocusMode(false);
-        return !(focusMode.equals(Parameters.FOCUS_MODE_INFINITY)
-                || focusMode.equals(Parameters.FOCUS_MODE_FIXED)
-                || focusMode.equals(Parameters.FOCUS_MODE_EDOF));
+        return getFocusMode(false).equals(Parameters.FOCUS_MODE_AUTO);
     }
 
     public void setZslEnable(boolean value) {
-- 
2.17.1

